files/wheelnav/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "wheelnav",
+  "repo": "softwaretailoring/wheelnav",
+  "files": {
+    "include": ["wheelnav.js", "wheelnav.min.js", "wheelnav.min.js.map"]
```

Comment:
Thank you for response!
I supposed that the robot reads the 'js/dist' path from package.json.
The update.json example contains the non-minified version, so I believed it is needed.